### ISSUE TITLE
result not reproduced by taking gradient wrt logit and flipping sign

### ISSUE THREAD

**Issue by avani17101** [tensorflow/tcav#124]
I had the same doubt as https://github.com/tensorflow/tcav/issues/83 https://github.com/tensorflow/tcav/issues/107 https://github.com/tensorflow/tcav/issues/41
I  flipped the sign for tcav calc (< changed to >)and took gradient wrt logit(unnormalized output of network, `self.pred`).
 Still, I am not getting the same tcav scores as when grad wrt loss and tcav calc sign being <

Why is this happening? @BeenKim ?

**Comment by avani17101** [comment]
So the output for importance of zebra in inception network when wrt loss and < sign 
![image](https://user-images.githubusercontent.com/33591258/146674427-9513e2cc-156b-4722-86ad-f5c4256db842.png)
whereas when wrt output and > sign I get this
![image](https://user-images.githubusercontent.com/33591258/146674440-43ade4ac-1a58-4a64-8d18-8227d2d9ed8e.png)
